migration/data/course_tables.sql

```diff
@@ -260,6 +260,7 @@ CREATE TABLE gradeable_component_data (
     gd_id integer NOT NULL,
     gcd_score numeric NOT NULL,
     gcd_component_comment character varying NOT NULL,
+    gcd_has_custom boolean NOT NULL, 
```

Comment:
Why is this necessary?

Reply:
To see if the custom mark is checked. Before, if the custom mark had no comment, it was treated as unchecked, and would automatically uncheck if the comment was removed. gcd_custom brings the structure of a custom mark closer to that of a regular mark.